src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+
+public class Formatter {
```

Comment:
Though the class name gives an idea about the responsibility of this class, still it is encouraged to mention it clearly using comments